src/pc/sdl2.c

```diff
@@ -24,9 +18,13 @@ SDL_Renderer* g_Renderer = NULL;
 SDL_AudioSpec g_SdlAudioSpecs = {0};
 SDL_AudioDeviceID g_SdlAudioDevice = {0};
 DebugSdl g_DebugSdl = DEBUG_SDL_NONE;
-static SDL_Texture* g_VramTex = NULL;
-static int g_LastVramTexTpage = -1;
-static int g_LastVramTexClut = -1;
+SDL_Texture* g_VramTex = NULL;
+int g_LastVramTexTpage = -1;
```

Comment:
Can we try to move some of this stuff inside `render_gl.c` and mark it as `static` again?

Reply:
g_WndWidth/Height are used by both renderers.
g_LastVramTexTpage/Clut are used by ResetPlatform as well as the gl renderer
g_VramTex/g_DebugSdl is used by sdl.c and gl.c
 Is there something in particular you want moved?

Reply:
mmh then I think it's fine for now as it is. I will think of a more structured solution as a separate PR